.github/workflows/build-and-publish-images.yaml

```diff
@@ -80,4 +80,5 @@ jobs:
           IMAGE_REPO: ${{ matrix.IMAGE_REPO }}
           BUILD_TYPE: ${{ inputs.build_type }}
           IMAGE_NAME: ${{ matrix.IMAGE_NAME }}
+          IMAGE_NAME_NO_RAPIDS_VERSION: ${{ matrix.IMAGE_NAME_NO_RAPIDS_VERSION }}
```

Comment:
This is only needed in the jobs that are pushing new manifests, because I'm not proposing also double-publishing the architecture-specific tags like `rapidsai/ci-conda:[25.08-]cuda12.8.0-ubuntu24.04-py3.10-arm64`.

Those tags can technically be pulled directly, but they're really just intended as implementation details of the non-architecture-specific manifests, e.g. `rapidsai/ci-conda:[25.08-]cuda12.8.0-ubuntu24.04-py3.10`